### ISSUE TITLE
FEATURE REQUEST: Being able to manage multiple Wireguard interfaces?

### ISSUE THREAD

**Issue by DaanSelen** [wg-easy/wg-easy#1082]
Excuse my hijacking of the bug report forum. I did 

I want to ask if its possible to manage multiple Wireguard interfaces, e.g: wg0, wg1, wg2 and their firewall rules with WG-Easy. That would be the step for our company to use this. If not, I would like to ask for a feature request.

I hope to hear from the project team!

**Comment by davide-acanfora** [comment]
I agree that it would be a really interesting feature to have. I won't hide the fact that lately I have started thinking about how to implement it and all the changes that would be needed to make it possible. Definitely something worth spending time on

**Comment by pheiduck** [comment]
Someone should came up to make a PR for that feature. :wink:

**Comment by DaanSelen** [comment]
I have almost no experience is JS, but it might be possible to: in WireGuard.js have the Wireguard Interface itself also as a variable. But then you need a way to have the persistence handled. And I am not familiar how wg-easy it does now, so I can't say how it would be best.
